src/promdens/promdens.py

```diff
@@ -111,24 +111,23 @@ def __post_init__(self):
 
         if self.envelope_type == 'gauss':
             self.equation = "exp(-2*ln(2)*(t-t0)^2/fwhm^2)*cos((omega+chirp*t)*t)"
-            extent = 2.4 * self.fwhm
+            extent = 2.4*self.fwhm
```

Comment:
Feel free to rename this to temporal_extent. (Although I guess this is half of that,  not sure how to express that )

Reply:
I forgot to do that, thanks. I will change it.